index.bs

```diff
@@ -4814,22 +4814,22 @@ Updating Registry Tables</h4>
 	(i.e. [[#correction-classes|Class 5 changes]])
 	can be made by re-publishing the [=technical report=] that contains the affected table,
 	without needing to satisfy any other requirements for the publication
-	(not even Working Group consensus, unless this is required by the [=registry definition=]).
+	(not even consensus, unless this is required by the [=registry definition=]).
 	Such [=registry changes=] do not trigger new [=Advisory Committee Reviews=],
 	nor Exclusion Opportunities,
 	and do not require verification via an [=update request=],
 	even for [=technical reports=] at maturities where this would normally be expected.
 	Such publications can be made
-	even in the absence of a [=Working Group=] chartered to maintain the registry
+	even in the absence of a group chartered to maintain the registry
 	when the [=custodian=] is another entity.
 
 	Note: The custodian is only empowered to make [=registry changes=].
-	If the Working Group establishing the registry wishes
+	If the [=chartered group|chartered=] or [=elected group=] establishing the registry wishes
```

Comment:
```suggestion
	If the [=group=] establishing the registry wishes
```

Reply:
Ok with dropping to group, but I don't think the hyperlinking helps

Reply:
So I'll go with this:
```suggestion
	If the group establishing the registry wishes
```